README.md

```diff
@@ -1,185 +1,268 @@
+Now use the following commands to auto-configure, build and install the `arpack-ng`.
 
-   Example programs for banded, complex, nonsymmetric, symmetric,
-   and singular value decomposition may be found in the directories
-   BAND, COMPLEX, NONSYM, SYM, SVD respectively.  Look at the README
-   file for further information.  To get started, get into the SIMPLE
-   directory to see example programs that illustrate the use of ARPACK in
-   the simplest modes of operation for the most commonly posed
-   standard eigenvalue problems.
+```bash
+sh bootstrap
+./configure
+make
+make check
+make install
+```
 
+> Note, this will install the library at standard location (which is recommmended). You may need root privilage.
 
-   Example programs for Parallel ARPACK may be found in the directory
-   PARPACK/EXAMPLES. Look at the README file for further information.
+> Unlike ARPACK, ARPACK-NG is providing autotools and cmake based build system and iso_c_binding support (which enables to call fortran subroutines natively from C or C++).
 
-   The following instructions explain how to make the ARPACK library.
+#### Using CMake
 
-4. Unlike ARPACK, ARPACK-NG is providing autotools and cmake based build
-   system and iso_c_binding support (which enables to call fortran 
-   subroutines natively from C or C++).
+You can install `ARPACK-NG` by using CMake. If you do not have Cmake, then please download the binary from `pip` using:
 
-Therefore, the classical commands should work as expected:
-   
-   
-    $ sh bootstrap
-    $ ./configure
-    $ make
-    $ make check
-    $ make install
+```bash
+python3 -m pip install cmake
+which cmake && cmake --version
+```
 
-Furthermore, ARPACK-NG now provides CMake functionality:
-   
-    $ mkdir build
-    $ cd build
-    $ cmake -D EXAMPLES=ON -D MPI=ON -D BUILD_SHARED_LIBS=ON ..
-    $ make
-    $ make install
-   builds everything including examples and parallel support (with MPI).
+After installing CMake you should use following lines of code in the terminal:
 
-To use arpack from CMake, use ARPACK::ARPACK target:
+Note: Make sure you are in `arpack-ng` directory obtained from github.
 
-    find_package(arpack-ng)
-    add_executable(main main.f)
-    target_include_directories(main PUBLIC ARPACK::ARPACK)
-    target_link_libraries(main ARPACK::ARPACK)
+```fortran
+  mkdir build
+  cd build
+  cmake -D EXAMPLES=ON -D MPI=ON -D BUILD_SHARED_LIBS=ON ..
+  make
+  make install
+```
 
-To use parpack from CMake, use PARPACK::PARPACK target:
+> Do not miss the `..` in line 3, as it denotes the location of source files.
 
-    find_package(arpack-ng)
-    add_executable(main main.f)
-    target_include_directories(main PUBLIC PARPACK::PARPACK)
-    target_link_libraries(main PARPACK::PARPACK)
+> The above script will builds everything including examples and parallel support (with MPI).
+
+### Using ARPACK-NG in Cmake project
+
+You can use arpack in your CMake builds by using `ARPACK::ARPACK` target. For example,
```

Comment:
Should add the `CMAKE_MODULE_PATH` env variable must be updated by the user (or `find_package` won't find `arpack-ng` cmake file).